help/en/html/apps/index.shtml

```diff
@@ -91,15 +82,28 @@
      
       <h2>JMRI Features and Tools</h2>
 
+<p>JMRI software consists of a large library of functions divided into (1) a
```

Comment:
Why not present these items as a numbered list? ``<ol>``

Reply:
Was just copying text from somewhere else.  Have changed to an unordered list which I think is appropriate here.